app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,30 @@
+class SessionsController < ApplicationController
+  skip_before_action :authorized, only: [:new, :create, :welcome]
+
+  def new
+  end
+
+  def create
```

Comment:
Is this method for logging in or creating an account? Could use either comments or a more descriptive method name to make it clear

Reply:
It was created from rails when we created the sessions controller